tests/codegen/fcode/scripts/functions.py

```diff
@@ -64,7 +64,9 @@ def f2(x, m=None):
 
 helloworld()
 
-# TODO add messages. for the moment there's a bug in Print
```

Comment:
why you remove this comment, did you check if the bug mentioned here is solved or not?
you did not mention the reason for removing this todo comment in your commit https://github.com/pyccel/pyccel/pull/687/commits/f806164cd0c1e21b55e63a3e1590461e6c7eee81
It will be useful to describe the reason in the commit, so anyone who reviews the code will understand your changes.

Reply:
we already discussed about it in this PR, https://github.com/pyccel/pyccel/pull/687#discussion_r561635957